go/src/koding/kites/config/konfig.go

```diff
@@ -135,6 +136,7 @@ func NewKonfig(e *Environments) *Konfig {
 		KontrolURL:         Builtin.Endpoints.Kontrol,
 		KloudURL:           Builtin.Endpoints.Kloud,
 		TunnelURL:          Builtin.Endpoints.TunnelServer,
+		RemoteURL:          Builtin.Endpoints.RemoteAPI,
```

Comment:
I agree, but I'd want to fix more than this:

- currently we mix public / private endpoint configuration - some of the urls are https://koding.com/remote.api and some https://127.0.0.1/api/credential
- I think we should keep the URLs separate in case we'd want in future to have each service in separate container / POD
- moreover we should add two URLs per serivce - one public (for koding users) and one private (for inter-communication)

So actually I wanted to even more separate the URLs than to group them. Do we care?